source/code-snippets/usage-examples/count.ts

```diff
@@ -0,0 +1,27 @@
+import { MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const movies = database.collection("movies");
+
+    const estimate = await movies.estimatedDocumentCount();
```

Comment:
we don't think this TS example is necessary since it is identical to the `.js` one and there is no added value from TS over plain JS for this use case

Reply:
Adding fix in [PR with similar comment](https://github.com/mongodb/docs-node/pull/233#discussion_r686873680)